modules/cnf-installing-numa-resources-operator-cli.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/cnf-numa-aware-scheduling.adoc
+
+:_content-type: PROCEDURE
+[id="cnf-installing-numa-resources-operator-cli_{context}"]
+= Installing the NUMA Resources Operator using the CLI
+
+As a cluster administrator, you can install the Operator using the CLI.
+
+.Prerequisites
+
+* Install the OpenShift CLI (`oc`).
+
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a namespace for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-namespace.yaml` file:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: openshift-numaresources
+----
+
+.. Create the `Namespace` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-namespace.yaml
+----
+
+. Create the operator group for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-operatorgroup.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: numaresources-operator
+  namespace: openshift-numaresources
+spec:
+  targetNamespaces:
+  - openshift-numaresources
+----
+
+.. Create the `OperatorGroup` CR:
++
+[source,terminal]
+----
+$ oc create -f nro-operatorgroup.yaml
+----
+
+. Create the subscription for the NUMA Resources Operator:
+
+.. Save the following YAML in the `nro-sub.yaml` file:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: numaresources-operator
+  namespace: openshift-numaresources
+spec:
+  channel: "4.10"
```

Comment:
for other supported version > 4.10 the channel should be adjusted accordingly. I've seen in other docs that they use a parameter for the version, maybe it can be applied here and in other similar places in the doc.

Reply:
We can use `stable-{product-version}` or `{product-version}`?

Reply:
I believe {product-version} should be good and keep it simple because the feature should work on any version >= 4.10